src/gossie/mapping.go

```diff
@@ -282,14 +282,16 @@ func (m *sparseMapping) Map(source interface{}) (*Row, error) {
 		if err != nil {
 			return nil, err
 		}
+		cp := make([]byte, 0, len(composite))
 		if len(composite) > 0 {
-			columnName = append(composite, packComposite(columnName, eocEquals)...)
+			cp = append(cp, composite...)
+			cp = append(cp, packComposite(columnName, eocEquals)...)
```

Comment:
If  composite is empty then the column name will end up empty - should this line be in the if block?